date/people.json

```diff
@@ -0,0 +1 @@
+[{"json_class":"Student","id":422,"name":"Gabriel","age":20,"parent_permission":true,"classroom":null},{"json_class":"Teacher","id":167,"name":"Vicente","age":30,"specialization":"vet"}]
```

Comment:
- [ ] It appears you are not making use of the files within the `date` directory. When I create books, people, and rentals, the files are saved in JSON files within the root path. Kindly remove this file and all the ones within the `date` folder to keep your code clean.